src/etl/ReportingETL.cpp

```diff
@@ -60,13 +61,29 @@ ReportingETL::insertTransactions(
         result.accountTxData.emplace_back(
             txMeta, sttx.getTransactionID(), journal);
         std::string keyStr{(const char*)sttx.getTransactionID().data(), 32};
+
+        ripple::uint256 hash{keyStr};
+        hashes.push_back(hash);
+
         backend_->writeTransaction(
             std::move(keyStr),
             ledger.seq,
             ledger.closeTime.time_since_epoch().count(),
             std::move(*raw),
             std::move(*txn.mutable_metadata_blob()));
+
+        Backend::Blob txBlob;
+        std::copy(raw->begin(), raw->end(), std::back_inserter(txBlob));
+        Backend::Blob mdBlob;
+        auto mdStr = txn.mutable_metadata_blob();
+        std::copy(mdStr->begin(), mdStr->end(), std::back_inserter(mdBlob));
+        transactions.push_back(
+            {txBlob,
+             mdBlob,
+             ledger.seq,
+             ledger.closeTime.time_since_epoch().count()});
     }
```

Comment:
This is a use after move bug, since we move from `raw` and `txn.mutable_metadata_blob()` a few lines above this, when we call `writeTransaction`. We need to make this copy before we call `writeTransaction`.

Also, I think this can be done in one line, like so (didn't test this):
```
transactions.push_back({
    {raw->data(),raw->size()},
    {txn.mutable_metadata_blob()->data(),txn.mutable_metadata_blob()->size()},
    ledger.seq,
    ledger.closeTime.time_since_epoch().count()}});
```